src/buttons/button.scss

```diff
@@ -15,7 +17,9 @@
   background-repeat: repeat-x;
   background-position: -1px -1px;
   background-size: 110% 110%;
-  border: 1px solid transparentize($black, 0.8);
+  // stylelint-disable-next-line primer/borders
+  border: $border-width $border-style transparentize($black, 0.8);
```

Comment:
> At the very least, I think we should standardize on using `rgba($color, $alpha)` rather than `transparentize($color, $alpha-offset)`.

Yeah, I find `rgba($black, 0.2)` much easier to read. Maybe because `transparentize($black, 0.8)` requires you to do some math first. I guess the only time we would need `transparentize()` is when we want to make a color that **already is** semi-transparent a little more transparent. But that's maybe a rare case we don't have to worry about.